core/data_migration_scripts/EE_DMS_Core_4_9_0.dms.php

```diff
@@ -404,15 +404,15 @@ public function schema_changes_before_migration()
 				QST_system varchar(25) DEFAULT NULL,
 				QST_type varchar(25) NOT NULL DEFAULT "TEXT",
 				QST_required tinyint(1) unsigned NOT NULL DEFAULT 0,
-				QST_required_text varchar(100) NULL,
+				QST_required_text text NULL,
 				QST_order tinyint(2) unsigned NOT NULL DEFAULT 0,
 				QST_admin_only tinyint(1) NOT NULL DEFAULT 0,
 				QST_max smallint(5) NOT NULL DEFAULT -1,
 				QST_wp_user bigint(20) unsigned NULL,
 				QST_deleted tinyint(2) unsigned NOT NULL DEFAULT 0,
 				PRIMARY KEY  (QST_ID),
 				KEY QST_order (QST_order)';
-        $this->_table_has_not_changed_since_previous($table_name, $sql, 'ENGINE=InnoDB');
+        $this->_table_is_changed_in_this_version($table_name, $sql, 'ENGINE=InnoDB');
```

Comment:
Is this so that _existing_ 4.9.0 sites when updated to 4.9.x will get the table updated?

Reply:
Hmmmm thinking about it it's not necessary. I originally thought it would help when someone updates from 4.9.x to this version, because we're telling it the DB changed. But on an update like that we're going to check all the database tables anyway. The methods `_table_has_not_changed_since_previous` and `_table_is_changed_in_this_version` really just affect migrations. And in this case, if someone's going through all the migrations, there's actually nothing changed in this table.

(The only case this could help is someone has been on 4.8.x, then upgrades to this version of 4.9.x; they'll get the changes to this database table. But I doubt there are very many users hanging onto a 4.8.x version; they're basically all on 3.1.x, and those ones are going to run the full gamet of migrations, and so won't benefit from this.)